packages/rolldown/tests/package.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "rolldown-tests",
+  "private": true,
+  "scripts": {
+    "go": "cross-env RUST_BACKTRACE=1 ROLLDOWN_TEST=1 vitest run --reporter verbose --hideSkippedTests"
```

Comment:
I was using `run`, but `run` is quite special in `pnpm` and causes some wired behaviors, so I change it to `go`.